forward_engineering/helpers/updateHelpers/udtHelper.js

```diff
@@ -24,24 +25,31 @@ const getRenameType = (renameData) =>
 	`${getAlterTypePrefix(renameData.keySpaceName)}."${renameData.udtName}" 
 	RENAME "${renameData.oldFieldName}" TO "${renameData.newFieldName}";`;
 
-const getDropUDT = (dropUDTData) => ([{
-	...scriptData,
-	script: `DROP TYPE IF EXISTS "${dropUDTData.keyspaceName}"."${dropUDTData.typeName}";`,
-	deleted: true,
-}]);
+const getDropUDT = (dropUDTData) => ([
+	AlterScriptDto.getInstance(
+		[`DROP TYPE IF EXISTS "${dropUDTData.keyspaceName}"."${dropUDTData.typeName}";`], 
```

Comment:
DDL templates are supposed to be in templates.js file. ddlProvider must've been used for filling templates with values